File: JAVA/secret_values.js
var secret_color = "blue";
var secret_number = 20;

function guess_secrets(color, number){
    color = prompt("Color ? ");
    number = prompt("Number ?");

    if(color == secret_color && number == secret_number){
        console.log("You are awesome! ");

    } else {
        console.log("Try again");
    }
}

guess_secrets();

File: PYTHON/secret_values.py
color = input("Color ? ")
number = int(input("Number ? "))

def guess_secrets(color, number):

    secret_color = "blue"
    secret_number = 20

    if color == secret_color and number == secret_number:
        print("Well fucking done! ")
    elif color == secret_color or number == secret_number:
        print("One out of Two! ")
    else: print("Too bad! ") 

guess_secrets(color, number)

